### DISCUSSION TITLE
ProLevel with Audirvana/DAC via BlackHole

### DISCUSSION THREAD

**Discussion by fredmaher1982** [ExistentialAudio/BlackHole#733] [General]
Hi. I just bought the ProLevel VU meter app and the site claims that it can be used to monitor any output on the Mac via BlackHole. Despite my best efforts, I cannot get it to work with my DAC. When I try to use multi-output or aggregate with Audirvana, it gives me the following error: "The sound device is already hogged by another application((null), pid = 0). Unable to start playing". Exclusive access is not selected in Audirvana. Anyone successfully set up the same or similar configuration as me via BlackHole? I'm using a Mac Mini M1. Thanks for any help you can give me.

**Comment by devinroth** [comment]
Are you on Ventura? If so, my new app Dipper (paid with free demo) will solve the problem. Give it a try.

https://apps.apple.com/app/dipper/id6450242673?mt=12

Devin

**Comment by fredmaher1982** [comment]
Many thanks, Devin, I'll check it out.
